resources/assets/templates/vote/voteItem.hbs

```diff
@@ -10,9 +10,15 @@
             <button class="text-info subscribe-btn"> Subscribe</button>
         </div>
         <div class="votes-item-details">
-            <span>{{ meta.comments}} Comments</span> <span>{{ meta.likes }} likes</span> <span>{{ meta.numberOfUniqueViews }} unique views</span>
-            <span>Finished at {{model.finished_at}}</span><span>{{model.views}} Views</span>
-
+            <span>{{ meta.comments}} Comments</span>
+            <span>{{ meta.likes }} likes</span>
+            <span>{{ meta.numberOfUniqueViews }} unique views</span>
+            {{#if model.finished_at}}
+            <span>Finished at {{model.finished_at}}</span>
+            {{else}}
+            <span>End date is not defined</span>
```

Comment:
Please replace 'End date is not defined' to 'currently in process', add style class with green text